documentation/release_notes.rst

```diff
@@ -8,6 +8,98 @@ The Intel® oneAPI DPC++ Library (oneDPL) accompanies the Intel® oneAPI DPC++/C
 and provides high-productivity APIs aimed to minimize programming efforts of C++ developers
 creating efficient heterogeneous applications.
 
+New in 2022.6.0
+===============
+News
+------------
+- `oneAPI DPC++ Library Manual Migration Guide`_ to simplify the migration of Thrust* and CUB* APIs from CUDA*. 
+- ``radix_sort`` and ``radix_sort_by_key`` algorithms residing in the ``oneapi::dpl::experimental::kt::esimd`` namespace
+  were moved into ``oneapi::dpl::experimental::kt::gpu::esimd`` namespace. 
+  The former namespace is now deprecated and will be removed in the future release.
+
+New Features
+------------
+- oneDPL code is now implemented on tag dispatching approach. 
```

Comment:
"tag dispatching" is a kind of oneDPL implementation details. I think we should not mention that in the release notes.